src/probnum/diffeq/_probsolve_ivp.py

```diff
@@ -33,7 +32,7 @@ def probsolve_ivp(
     step=None,
     diffusion_model="dynamic",
 ):
-    r"""Solve initial value problem with Gaussian filtering and smoothing.
+    r"""Solve an initial value problem with a probabilistic ODE solver, i.e. with Gaussian filtering and smoothing.
```

Comment:
Instead of "probabilistic ODE solver", should we specify "filtering-based" or something now that we also have the sampling/perturbation-based probabilistic ODE solvers?

Reply:
My idea here was to nudge the narrative slowly towards filtering-based solvers being **the** probabilistic ODE solver :)

Reply:
Agreed, the `i.e., ...` - part suggests that `probabilistic Ode solver == filtering-based ODE solver`.

Reply:
As much as I like this idea, I'm not sure how people outside of our group agree with this assessment?

But apart from that wider discussion: The solvers in `perturbed` are also probabilistic ODE solvers, and I imagine that tutorials and the documentation will describe them as such. To say that `probsolve_ivp` solves an IVP with "a probabilistic ODE solver" seems to me a bit misleading then, since it might suggest that sampling-based solvers could be used here.

Reply:
Oh, @pnkraemer , your comment wasn't displayed, when I wrote my answer.
I see your point as well, it just depends on the message we want to send. As a package which offers different kinds of probabilistic ODE solvers as well, this may be ... inconsistent I guess.

Reply:
Whoops, was that "i.e. ..." part there all the time? Then my brain somehow completely ignored that. With that addition everything is totally fine, sorry for the discussion I started!

Reply:
Hmmm okay, I change it to 
```
    r"""Solve an initial value problem with a (filtering-based) probabilistic ODE solver.
```
for now (I understand your points), but the last word has not been spoken here! ;)